docs/index.rst

```diff
@@ -73,6 +73,6 @@ NVIDIA DALI Documentation
    :hidden:
    :caption: Reference
 
-   Release Notes <https://docs.nvidia.com/deeplearning/dali/release-notes.html>
+   Release Notes <https://docs.nvidia.com/deeplearning/dali/release-notes/index.html>
```

Comment:
We have the same link in L15 of this file. I don't what about the https://docs.nvidia.com/deeplearning/dali/archives.html from conf.py

Reply:
Done